docker/README.md

```diff
@@ -0,0 +1,22 @@
+# Suricata (4.0.5) in Docker
+
+The open-source Intrusion Detection System, in Docker, built from source with **suricata-update**, **Hyperscan**, **GeoIP** and **Lua** support.
+
+## Usage
+
+```bash
+docker build suricata:latest .
+```
+
+```bash
+docker run -d \
+    --name suricata \
+    --privileged \
+    --network host \
+    --cap-add NET_ADMIN \
```

Comment:
I'm a bit rusted on docker side but if you run in privileged mode do you really need to add the capnet ?

Reply:
Correct! I've amended the doc.